### PR TITLE
Update the urlllib3 package dependency to make it compatible with v2

### PR DESCRIPTION
**Pull request by lingcoder**
### All Submissions:

* [x] Contributions should target the `dev` branch. Did you create your branch from `dev`?
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you installed `pre-commit` with `pip3 install pre-commit` and set up hooks with `pre-commit install`?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by joein** [comment]
I believe it won't work, since `^1.26.4` is equal to `>=1.26.4,<2.0.0`

Instead, it should be `urllib3 = ">=1.26.14,<3"`

**Comment by lingcoder** [comment]
> I believe it won't work, since `^1.26.4` is equal to `>=1.26.4,<2.0.0`
> 
> Instead, it should be `urllib3 = ">=1.26.14,<3"`



> 我相信这是行不通的，因为`^1.26.4`等于`>=1.26.4,<2.0.0`
> 
> 相反，它应该是`urllib3 = ">=1.26.14,<3"`

It's truly what I committed. `urllib3 = ">=1.26.14,<3`

**Comment by joein** [comment]
To enable installation of v2+, this dependency should be changed to ">1.26.4,<3", do you want to do it or should I take over?

**Comment by lingcoder** [comment]
> To enable installation of v2+, this dependency should be changed to ">1.26.4,<3", do you want to do it or should I take over?

Oh, my apologies, I finally understand where I went wrong. I have submitted the latest PR, and it would be my honor to contribute code to qdrant.

File: pyproject.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ numpy = [
 pydantic = ">=1.10.8"
 grpcio = { version = ">=1.41.0", allow-prereleases = true }
 grpcio-tools = ">=1.41.0"
-urllib3 = "^1.26.14"
+urllib3 = ">1.26.4,<3"
```
Comment (joein) [inline pyproject.toml:27]:
```suggestion
urllib3 = ">=1.26.4,<3"
```

**Review by joein — CHANGES_REQUESTED** [review]

**Review by joein — APPROVED** [review]

**Pull request merged**